tests/xammac_tests/xammac_tests.csproj

```diff
@@ -17,7 +17,7 @@
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
     <OutputPath>bin\x86\Debug</OutputPath>
-    <DefineConstants>__UNIFIED__;DEBUG;MONOMAC;XAMCORE_2_0</DefineConstants>
+    <DefineConstants>__MACOS__;__UNIFIED__;DEBUG;MONOMAC;XAMCORE_2_0;XAMMAC_TESTS</DefineConstants>
```

Comment:
Just a nitpick - should the changes to this project file be included in this PR?

Reply:
Yes, this is intentional: some of the other changes files are already included in xammac_tests.csproj, and since those changed files now need tests/common/Configuration.cs, this csproj now includes Configuration.cs, and Configuration.cs needed to change to make it build as part of xammac_tests.csproj.

That said, not *all* changes in the csproj are required; I'll remove those that aren't.